game-core/src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -102,6 +107,35 @@ public static GameData loadGame(final InputStream is) throws IOException {
     }
   }
 
+  private static void promptToLoadNewerSaveGame(final Version version) throws IOException {
+    // we can still load it because our engine is compatible, however this save was made by a
+    // newer engine, so prompt the user to upgrade
+    // this is needed because a newer client might depend on variables that are part of the new
+    // save but will be stripped by the old client. When the old client re-saves the data, the
+    // new client will load the save game and be in odd state.
+    final String messageString =
+        "This save was made by a newer version of TripleA."
+            + "\nHowever, because the first version number is the same as your current version, "
+            + "we can still open the savegame. There is a possibility that if you save this game "
```

Comment:
> There is a possibility that if you save this game "
            + "and then open it up in the version that created it, the game will be in an "
            + "incorrect state.

We can probably state this more simply. Perhaps something like:

```
Playing newer saves with an older engine can lead to unpredictable problems. It is recommended that you upgrade to the latest version of TripleA before playing this save game. To download the latest version of TripleA, Please visit UrlConstants.DOWNLOAD_WEBSITE"
+ \n\nDo you wish to continue and open this save with your current 'old' version?";
```

I think there might be a usability problem here as well because the URL cannot be copy/pasted and a user has to type it in manually. We can make the link clickable, I suspect that would be the best UX. @trevan is that something you would be willing to do?

"JEditorPaneWithClickableLinks" is the class that enables messages with clickable links.

Reply:
I've updated the text with what you wrote.  I haven't made the link clickable, though.